.github/workflows/publish-packages-1.0.yml

```diff
@@ -16,6 +16,9 @@ on:
   schedule:
     - cron: '0 0 * * *' # once in a day at 00:00
 
+permissions:
```

Comment:
I am fully support changes, but I do not have yet time to verify changes.
@trask, in this PR you are allowing only read permissions, under the hood [otel-bot is used](https://github.com/open-telemetry/opentelemetry-dotnet/blob/309ed2e7a8673b9ddb0d901a492c3eb52d3c27ac/.github/workflows/automation.yml):

What is more around line 125 GH API is called to create release. I am not fully familiar with GH permissions, do you think that it will be working without any problems?

Reply:
otelbot has its own token, so these permissions don't affect it